AWSTranscribeStreaming/AWSTranscribeStreamingService.h

```diff
@@ -186,9 +186,9 @@ For example, set the default service configuration in `- application:didFinishLa
                                                                                                         identityPoolId:@"YourIdentityPoolId"];
         AWSServiceConfiguration *configuration = [[AWSServiceConfiguration alloc] initWithRegion:AWSRegionUSWest2
                                                                              credentialsProvider:credentialsProvider];
-        WebSocketProvider = provider [[WebSocketProvider alloc] init;
+        NativeWebSocketProvider  webSocketProvider =  [[NativeWebSocketProvider alloc] init;
 
-        [AWSTranscribeStreaming registerTranscribeStreamingWithConfiguration:configuration forKey:@"USWest2TranscribeStreaming" provider: provider];
+        [AWSTranscribeStreaming registerTranscribeStreamingWithConfiguration:configuration forKey:@"USWest2TranscribeStreaming" webSocketProvider: webSocketProvider];
```

Comment:
nit: consistent spacing between colon and argument: `webSocketProvider:webSocketProvider`